SequentialClassifier.ipynb

```diff
@@ -0,0 +1,245 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **BOLT Sequential Classifier**\n",
+    "The sequential classifier is a plug-and-play classification model that leverages information from sequences of user interactions. These interactions can be anything from watching a movie on a streaming platform, to adding an item to cart on an online shop, to financial transactions. This makes the classifier especially useful for tasks like contextualized recommendations as well as fraud detection."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### **0. Install Packages**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip3 install thirdai\n",
+    "!pip3 install pandas"
```

Comment:
Should we fix a version number for these installations?

Reply:
GREAT idea